firefox-ios/l10n-screenshots.sh

```diff
@@ -34,7 +34,7 @@ for lang in $LOCALES; do
     echo "$(date) Snapshotting $lang"
     mkdir "l10n-screenshots/$lang"
     fastlane snapshot --project firefox-ios/Client.xcodeproj --scheme L10nSnapshotTests \
-        --number_of_retries 0 \
+        --number_of_retries 3 \
```

Comment:
I can't tell if the default value has changed or not by reading the documentation. Let's hard code this value.

Reply:
👀 `testHistoryTableContextMenu` failed and wasn't retried. Looking.
https://app.bitrise.io/build/bc7e1954-3780-4157-afeb-51e10ad9ff3d?tab=log

Reply:
This option is ignored now by fastlane. We should use set the retry opion to off from xcode and to do that, we need to create a test plan for the L10n schema where we can configure that. I have done that locally, forced a test to fail, and there are no retries.
Please try that and let me know in case you want a hack session to do that together.

Reply:
I have created a `L10nSnapshotTests` test plan and have the schema to use the test plan.